### PR TITLE
docs(forms): use `Validators.email` to validate email field

### PR DESCRIPTION
**Pull request by Vergil333**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Email field was validated to not be empty but not for correct email format.

Issue Number: N/A

## What is the new behavior?
Update shows how to use multiple validators for single field and how to properly validate email.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
N/A

### CODE REVIEW

File: adev/src/content/tutorials/learn-angular/steps/18-forms-validation/README.md [hunk 1]
```diff
@@ -23,12 +23,14 @@ export class AppComponent {}
 
 <docs-step title="Add validation to form">
 
-Every `FormControl` can be passed the `Validators` you want to use for validating the `FormControl` values. For example, if you want to make the fields for the `profileForm` required then use `Validators.required`. Update the `name` and `email` `FormControl` to be required:
+Every `FormControl` can be passed the `Validators` you want to use for validating the `FormControl` values. For example, if you want to make the `name` field in `profileForm` required then use `Validators.required`.
+For the `email` field, we also do not want it to be empty and it has to be in correct email format, so we use `Validators.email`. To combine it with `Validators.required`, you pass them both in an array.
```
[the remark this answers is not in the document]
Comment (Vergil333) [reply adev/src/content/tutorials/learn-angular/steps/18-forms-validation/README.md]: agree, looks better. Updated.

**Comment by sonukapoor** [comment]
@Vergil333 LGTM. Can you just squash the commits and make sure the commit message is: `docs(forms): use Validators.email to validate email field`?

**Comment by sonukapoor** [comment]
You are doing great @Vergil333 - We appreciate your contribution 👍

**Review by sonukapoor — APPROVED** [review]

**Review by JeanMeche — APPROVED** [review]
LGTM, thank you for helping us improve the docs !

**Comment by JeanMeche** [comment]
Can you take a look at the failed linting job, the commit message need to follow our [linting rules](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
), thank you.

**Comment by Vergil333** [comment]
@JeanMeche ok, I think I understand now proper linting format of commit messages. Should be fine now :)

**Comment by atscott** [comment]
This PR was merged into the repository by commit 216199dd4df7df80a5e1c6342942fc78b80b4607.

**Pull request closed without merging**